### ISSUE TITLE
[Shaman] Maelstrom Weapon

### ISSUE THREAD

**Issue by Veretos** [tobmaps/TrinityCore#23]
Enhancement Shamans "Maelstrom Weapon" spell overprocs like crazy

**Comment by tobmaps** [comment]
i think it's ok, this proc chance is based at info from http://www.wowwiki.com/Maelstrom_Weapon

**Comment by Veretos** [comment]
I have done a 1 minute test with a shaman that had 2 2.6 weapons and the results were: 65 procs in that one minute test. According to wowiki it should be around 25

**Comment by tobmaps** [comment]
it has 12 ppm at 5 rank into our DB (TDB), maybe your DB has wrong data or ppm doesn't works correctly for this spell :S

**Comment by Veretos** [comment]
Weird I am using TDB aswell. Could you tell me which table is the proc part of?

**Comment by tobmaps** [comment]
select \* from spell_proc_event where entry in (51528,51529,51530,51531,51532);

**Comment by Veretos** [comment]
Yes you are right it set to 12...I wonder why does it proc so much then. Do you think you can try it on your realm?

**Comment by Veretos** [comment]
Hi, I did few more tests and it seems that the spell procs form Windfury Weapon.

**Comment by Wattiqelumes** [comment]
Bump, confirmed.
That's why it overproccs. It's triggered by Windfury Weapon.

**Comment by Wattiqelumes** [comment]
Bump. Really need a fix for this. :s

**Comment by tobmaps** [comment]
i don't know if it bug or no. Need some info from trusted source

**Comment by Wattiqelumes** [comment]
I've just got told by a friend it SHOULD procc from Windfury Weapon, but the procc chance of it is too high. Shouldn't be 100% on Windfury.
In the tooltip it says ''You have a CHANCE to...''
